app/views/questionnaire.py

```diff
@@ -530,6 +485,15 @@ def _build_template(current_location, context=None, template=None, display_navig
     return _render_template(context, current_location.block_id, front_end_navigation, metadata_context, current_location, previous_url, template)
 
 
+def _get_front_end_navigation(answer_store, completed_blocks, current_location, metadata):
+    if not current_location.is_interstitial():
```

Comment:
Seems like a double check as well as we do
```if block_json['type'] == 'questionnaire' or block_json['type'] == 'interstitial' else None```
further down. We should be able to get rid of one